### PR TITLE
Resolved Depreciation Warning Issue

### PR DESCRIPTION
**Pull request by AnushkaShreyam**
- Added pyproject.toml for project metadata and build automation.
- Updated README.md with revised setup and usage instructions.
Related to Issue #1 
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,18 @@ You can install Ersilia Client directly from the source code:
...
 git clone https://github.com/ersilia-os/ersilia-client.git
 cd ersilia-client
-python setup.py install -e .
+pip install -e .
 ```
 
+Note: If you encounter any issues with the installation, you can try the following command:
+
+```bash
+pip install -e . --config-settings editable_mode=compat
```
Comment (DhanshreeA) [inline README.md]: I don't think we should encourage this. This is a very simple package and should technically work without a legacy fallback.

File: setup.py
Comment (DhanshreeA) [inline setup.py:1]: I think we can directly remove the `setup.py` file.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+[project]
+name = "ersilia-client"  
+version = "0.1.0"
+description = "Interact with served Ersilia models through a simple Python client"
+authors = [
+    { name = "Ersilia Open Source Initiative", email = "hello@ersilia.io" }
+]
+readme = "README.md"
+requires-python = ">=3.7"
```
Comment (DhanshreeA) [inline pyproject.toml]: Let's try to support Python 3.8 and above, Python 3.7 is slowly being phased out.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+license = { text = "MIT" }
+keywords = ["ersilia", "client", "open-source"]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent"
+]
+dependencies = [
+    "requests",
```
Comment (DhanshreeA) [inline pyproject.toml]: Let's pin the dependency versions.

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AnushkaShreyam) [reply README.md]: Sure.

**Review by DhanshreeA — APPROVED** [review]

**Review by DhanshreeA — APPROVED** [review]

**Pull request merged**